test/CMakeLists.txt

```diff
@@ -16,13 +16,14 @@ function(add_base64_test TEST_NAME)
     foreach(SRC_FILE ${ARGN})
         list(APPEND SRC_FILES "${SRC_FILE}")
     endforeach()
-    
+
     add_executable(${TEST_NAME} ${SRC_FILES})
     target_link_libraries(${TEST_NAME} PRIVATE base64)
-    
+
     add_test(NAME ${TEST_NAME}
         COMMAND ${TEST_NAME}
     )
+    install(TARGETS ${TEST_NAME} DESTINATION bin)
```

Comment:
What is the use case for installing test executables?

Reply:
I have no problem with this, because the tests are not installed when test creation is disabled.